packages/owl-bot/test/copy-code.ts

```diff
@@ -69,6 +70,44 @@ Copy-Tag: ${copyTag}
     );
   });
 
+  it('finds pull request in cache with copy tag', async () => {
+    const octokit = await fakeOctokit();
+    const copyTag = copyTagFrom('some-api/.OwlBot.yaml', 'abc123');
+    await octokit.pulls.create({
+      body: `blah blah blah
+Source-Link: https://github.com/googleapis/googleapis/abc123
+Copy-Tag: ${copyTag}
+`,
+    });
+    const destRepo: AffectedRepo = {
+      yamlPath: 'some-api/.OwlBot.yaml',
+      repo: githubRepoFromOwnerSlashName('googleapis/spell-checker'),
+    };
+    const cache = newRepoHistoryCache();
+    assert.strictEqual(
+      true,
+      await copyExists(
+        octokit as unknown as OctokitType,
+        destRepo,
+        'abc123',
+        1000,
+        cache
+      )
+    );
+
+    // Confirm its in the cache.
+    assert.strictEqual(
```

Comment:
These asserts look similar, do we need both?

Reply:
The first assert is optional.  If it were to fail, it be more obvious what had failed, so I left it in.